gui/src/euRun.cc

```diff
@@ -174,7 +174,54 @@ void RunControlGUI::OnReceive(const eudaq::ConnectionInfo &id,
     EmitStatus("EVENT", status->GetTag("EVENT"));
     EmitStatus("FILEBYTES", to_bytes(status->GetTag("FILEBYTES")));
   } else if (id.GetType() == "Producer") {
-    if (id.GetName() == "TLU" || id.GetName() == "miniTLU") {
+ if (id.GetName() == "caliceahcalbif") {
+      EmitStatus("TRIG", status->GetTag("TRIG"));
+      EmitStatus("PARTICLES", status->GetTag("PARTICLES"));
+      // EmitStatus("TIMESTAMP", status->GetTag("TIMESTAMP"));
```

Comment:
Yes, we are running without TLU, but with the AHCAL BIF which is a modification of the miniTLU that runs in slave mode.

I remove the dead code.